db/seeds.rb

```diff
@@ -1,125 +1,185 @@
 
-#breakfast items
+path = 'app/assets/images/'
+
+# #breakfast items ----------------------------------------------------------
 Item.create(title: "Bagel Sandwich",
-            description: "Toasted bagel, fried egg, and bacon",
-            price: 750,
-            image: "bagel_sandwich.jpg")
+		description: "Toasted bagel, fried egg, and bacon",
+		price: 750,
+		image: open(path + "bagel_sandwich.jpg"))
 
 CategoryItem.create(item_id: 1, category_id: 1)
 
 Item.create(title: "Bacon and Egg Cups",
-            description: "Bacon cups filled with egg",
-            price: 1100,
-            image: "bacon_and_egg_cups.jpg")
+		description: "Bacon cups filled with egg",
+		price: 1100,
+		image: open(path + "bacon_and_egg_cups.jpg"))
 
 CategoryItem.create(item_id: 2, category_id: 1)
 
 Item.create(title: "Bacon, Egg, and Cheddar Scones",
-            description: "Fresh cheddar cheese mixed with bacon and egg",
-            price: 1500,
-            image: "scones.jpg")
+		description: "Fresh cheddar cheese mixed with bacon and egg",
+		price: 1500,
+		image: open(path + "scones.jpg"))
 
 CategoryItem.create(item_id: 3, category_id: 1)
 
 #brunch items --------------------------------------------------------------
 Item.create(title: "Casserole",
-            description: "Delicious casserole filled with bacon",
-            price: 1200,
-            image: "casserole.jpg")
+		description: "Delicious casserole filled with bacon",
+		price: 1200,
+		image: open(path + "casserole.jpg"))
 
 CategoryItem.create(item_id: 4, category_id: 2)
 
 Item.create(title: "Asparagus Mushroom Bacon Quiche",
-            description: "Nothing better than a quiche with bacon",
-            price: 1100,
-            image: "quiche.jpg")
+		description: "Nothing better than a quiche with bacon",
+		price: 1100,
+		image: open(path + "quiche.jpg"))
 
 CategoryItem.create(item_id: 5, category_id: 2)
 
 Item.create(title: "Grilled Cheese French Toast With Bacon",
-            description: "Grilled cheese.  French toast.  Bacon.  Nuff said",
-            price: 1500,
-            image: "grilled_cheese.jpg")
+		description: "Grilled cheese.  French toast.  Bacon.  Nuff said",
+		price: 1500,
+		image: open(path + "grilled_cheese.jpg"))
 
 CategoryItem.create(item_id: 6, category_id: 2)
 
 #Lunch ----------------------------------------------------------------
 
 Item.create(title: "Bacon-Wrapped Jalapeno Poppers",
-            description: "Super spicy!",
-            price: 1000,
-            image: "pepper.jpg")
+		description: "Super spicy!",
+		price: 1000,
+		image: open(path + "pepper.jpg"))
 
 CategoryItem.create(item_id: 7, category_id: 3)
 
 Item.create(title: "Bacon Wrapped Chicken",
-            description: "Farm raised chicken wrapped in bacon, and grilled to perfection",
-            price: 1500,
-            image: "bacon_wrapped_chicken.jpg")
+		description: "Farm raised chicken wrapped in bacon, and grilled to perfection",
```

Comment:
Line is too long. [81/80]<br>Align the elements of a hash literal if they span more than one line.<br>Tab detected.